src/core/core.scale.js

```diff
@@ -359,6 +359,14 @@ export default class Scale extends Element {
     return this.options.labels || (this.isHorizontal() ? data.xLabels : data.yLabels) || data.labels || [];
   }
 
+  /**
+   * @return LabelItem[]
+   */
+  getLabelItems() {
+    const items = this._labelItems || (this._labelItems = this._computeLabelItems());
```

Comment:
Don't think this is can be done, the computeLabelItems expects the chartArea in some paths that will break now

Reply:
Agreed, I made the chartArea optional on getLabelItems() and I'm always passing it through from drawLabels()